core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -36,6 +36,7 @@ package object config {
 
   private[spark] val SPARK_RESOURCE_COUNT_POSTFIX = ".count"
   private[spark] val SPARK_RESOURCE_DISCOVERY_SCRIPT_POSTFIX = ".discoveryScript"
+  private[spark] val SPARK_RESOURCE_ADDRESSES_POSTFIX = ".addresses"
```

Comment:
I dont think you will need this anymore in this pr, I will add with the driver one